src/index.html

```diff
@@ -15,8 +15,100 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-links">
+          <li class="nav-link-item">
+            <a
+              class="nav-link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-link-item">
+            <a
+              class="nav-link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-link-item">
```

Comment:
There should be an empty line between multiline sibling blocks of HTML to add some 'air' and simplify reading. However, make sure not to add empty lines between parent and child elements.